web/src/main/java/org/cbioportal/web/mixin/MutSigMixin.java

```diff
@@ -0,0 +1,16 @@
+package org.cbioportal.web.mixin;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class MutSigMixin {
+
+    @JsonIgnore
+    private Integer cancerStudyId;
+    @JsonProperty("studyId")
+    private String cancerStudyIdentifier;
+    @JsonIgnore
+    private Integer numbasescovered;
```

Comment:
Because in the database the column name doesn't use underscore between words for this column, and I prefer to keep model property names in a way that it would match with database columns automatically if we use an ORM tool in the future.